cdtb/export.py

```diff
@@ -477,8 +477,13 @@ def filter_path(path):
         for path, wad in exporter.wads.items():
             if path.endswith('.wad.client'):
                 wad.files = [wf for wf in wad.files if filter_path(wf.path)]
+                language_match = re.match(r"\.(.._..)\.wad\.client$", path)
                 for wf in wad.files:
-                    wf.path = f"game/{wf.path}"
+                    if language_match is not None:
```

Comment:
This condition can be factorized. This is a bit cleaner and would be easier to extend if we have to introduce more special subdirs.
```py
            if path.endswith('.wad.client'):
                if lang_match := re.match(r"\.(.._..)\.wad\.client$", path):
                    subdir = f"game/{lang_match.group(1).lower()}"
                else:
                    subdir = "game"
                wad.files = [wf for wf in wad.files if filter_path(wf.path)]
                for wf in wad.files:
                    wf.path = f"{subdir}/{wf.path}"
```